api/app/Http/Controllers/PostController.php

```diff
@@ -64,10 +64,10 @@ public function store(Request $request)
         if ($post->is_job){
             $post->form_id = $request->input('job_form');
         }
-        
         $post->user_id = auth()->user()->id;
         $post->save();
-        if ($post->is_job){
+
+        if ($post->is_job && (auth()->user()->id == 2 || auth()->user()->id == 4)){
```

Comment:
ofc cia butu geriau metoda useriui padaryti is user ten koks student, nes as pvz nezinau kas tas 2, however mes sita projekta nelabai jau tavrkysim tai tegul buna cia tik side komentaras